### PR TITLE
update readme and package script

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ## Installation
 
 ```bash
-$ yarn install
+$ yarn install && yarn add @tonomy/tonomy-id-sdk@development
```
Comment (theblockstalk) [inline README.md:10]: missing `--no-lockfile` flag

Comment (sadiabbasi) [reply README.md:10]: for yarn it's working without --no-lockfile otherwise it throw error

**Pull request merged**